apps/agentstack-sdk-py/src/agentstack_sdk/a2a/extensions/services/mcp.py

```diff
@@ -39,7 +39,7 @@ class StdioTransport(pydantic.BaseModel):
 class StreamableHTTPTransport(pydantic.BaseModel):
     type: Literal["streamable_http"] = "streamable_http"
 
-    url: pydantic.AnyHttpUrl
+    url: str
```

Comment:
The problem is that client can provide `{platform_url}/foobar` like in other url fields.